ibm_catalog.json

```diff
@@ -807,7 +807,7 @@
           "dependencies": [
             {
               "name": "deploy-arch-ibm-vpc",
-              "description": "Configure the VPC instance and subnets where the OpenShift cluster will be deployed. ",
+              "description": "Configure the VPC instance and subnets where the OpenShift cluster will be deployed.  If not enabled, you must provide an existing VPC, if any. ",
```

Comment:
This is not what we decided to do - we decided to not make this optional, but a required dependency